support/logging/types.go

```diff
@@ -7,15 +7,32 @@
 package logging
 
 import (
-	"github.com/edgexfoundry/edgex-go/support/domain"
+	"fmt"
+	"os"
+	"strconv"
+
+	support_domain "github.com/edgexfoundry/edgex-go/support/domain"
 )
 
 const (
-	applicationName    = "support-logging"
-	defaultPort        = 48061
-	defaultPersistence = PersistenceFile
+	applicationName = "support-logging"
+	defaultPort     = 48061
+	//defaultPersistence = PersistenceFile
+	defaultPersistence = PersistenceMongo
 	defaultLogFilename = "support-logging.log"
 
+	defaultMongoDB             = "logging"
+	defaultMongoCollection     = "logEntry"
+	defaultMongoURL            = "127.0.0.1"
+	defaultMongoPort           = 27017
+	defaultMongoConnectTimeout = 5000
+	defaultSocketTimeout       = 5000
+	defaultMongoUsername       = "logging"
+	defaultMongoPassword       = "password"
+
+	envMongoURL  = "EXPORT_CLIENT_MONGO_URL"
```

Comment:
Export client? I guess it should be `SUPPORT_LOGGING`?

Reply:
Right! Fixed